src/main.js

```diff
@@ -59,20 +77,75 @@ function displayIdeas() {
             </div>
         </section>
     `
+  };
+};
 
-    }
+function clearInputs() {
+  titleInput.value = "";
+  bodyInput.value = "";
+  removeClickableHoverEffects(saveButton, saveButton);
+};
+
+function enableButton(event) {
+  if (titleInput.value !== "" && bodyInput.value !== "") {
+    saveButton.disabled = false;
+    addClickableHoverEffects(saveButton, saveButton);
+  } else {
+    saveButton.disabled = true;
+  };
+};
+
+function deleteIdea(event) {
+  remove();
+  displayIdeas();
+};
+
+function alterIdea(event) {
+  if (event.target.classList.contains(`x-button`)) {
+    deleteIdea();
+  } else if (event.target.classList.contains(`favorite`)) {
+    toggleStar();
+  };
+};
+
+function remove() {
+  for (var i = 0; i < ideas.length; i++) {
+    if (event.target.id === `${ideas[i].id}`) {
```

Comment:
Interesting that you are interpolating right here - could you not just have `ideas[i].id` ? What is the case for interpolation here?

Reply:
> Interesting that you are interpolating right here - could you not just have `ideas[i].id` ? What is the case for interpolation here?

It didn't when just doing the ideas[i].id.... I think we had this issue in the last project with it needing to be a number and not a string to match correctly